src/tests/test_common/homestore_test_common.hpp

```diff
@@ -459,5 +329,168 @@ class HSTestHelper {
             std::move(fut).thenValue(on_complete);
         }
     }
+
+private:
+    void do_start_homestore(bool fake_restart = false, bool init_device = true, uint32_t shutdown_delay_sec = 5) {
+        auto const ndevices = SISL_OPTIONS["num_devs"].as< uint32_t >();
+        auto const dev_size = SISL_OPTIONS["dev_size_mb"].as< uint64_t >() * 1024 * 1024;
+        auto num_threads = SISL_OPTIONS["num_threads"].as< uint32_t >();
+        auto num_fibers = SISL_OPTIONS["num_fibers"].as< uint32_t >();
+        auto is_spdk = SISL_OPTIONS["spdk"].as< bool >();
+
+        if (fake_restart) {
+            // Fake restart, device list is unchanged.
+            shutdown_homestore(false);
+            std::this_thread::sleep_for(std::chrono::seconds{shutdown_delay_sec});
+        } else if (SISL_OPTIONS.count("device_list")) {
+            // User has provided explicit device list, use that and initialize them
+            auto const devs = SISL_OPTIONS["device_list"].as< std::vector< std::string > >();
+            for (const auto& name : devs) {
+                // iomgr::DriveInterface::emulate_drive_type(name, iomgr::drive_type::block_hdd);
+                m_token.devs_.emplace_back(name, homestore::HSDevType::Data);
```

Comment:
Same for the real drive
```
 token.devs_.emplace_back(name,
                                             token.devs_.empty()
                                                 ? homestore::HSDevType::Fast
                                                 : homestore::HSDevType::Data); // First device is fast
```

Reply:
Yes let me change this one back. Good catch